sharktank/conftest.py

```diff
@@ -329,6 +343,14 @@ def model_artifacts(request: FixtureRequest) -> dict[str, str]:
     model_path["llama3_70b_f8_model_path"] = set_fixture_from_cli_option(
         request, "--llama3-70b-f8-model-path", "llama3_70b_f8_model"
     )
+    model_path["llama3_1_405b_instruct_f4_model_path"] = set_fixture_from_cli_option(
+        request,
+        "--llama3-1-405b-instruct-f4-model-path",
+        "llama3_1_405b_instruct_f4_model_path",
+    )
+    model_path["llama3_1_405b_tokenizer_path"] = set_fixture_from_cli_option(
+        request, "--llama3-1-405b-tokenizer-path", "llama3_1_405b_tokenizer_path"
+    )
```

Comment:
This is quickly getting out of hand. We need to change this to not continue with the combinatorial explosion.

Reply:
Maybe we can have a single CLI option like `--sharktank-test-data-path` for the test files and expect a standard directory structure inside across all machines.